src/main/java/Duke.java

```diff
@@ -1,10 +1,56 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        String hLine = "____________________________________________________________\n";
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(hLine + logo + " Hello! I'm Duke\n What can I do for you?\n" + hLine);
+        String echo = "";
+        Task[] tasks = new Task[100];
+        int listSize = 0;
+        Scanner sc = new Scanner(System.in);
+        while (!echo.equals("bye")) {
+            echo = sc.nextLine().trim();
+            if (echo.equals("list")) {
+                System.out.println(hLine + " Here are the tasks in your tasks:");
+                for (int i = 0; i < listSize; i++) {
+                    System.out.println(" " + (i + 1) + "." + tasks[i]);
+                }
+                System.out.println(hLine);
+            } else if (echo.length() > 5 && echo.substring(0, 4).equals("done")) {
+                int selected = Integer.valueOf(echo.substring(5)) - 1;
+                tasks[selected].markAsDone();
+                System.out.println(hLine + " Nice! I've marked this task as done: ");
+                System.out.println("   " + tasks[selected] + "\n" + hLine);
+            } else if (!echo.equals("bye")) {
+                String tmp = echo.split("\\s+")[0];
```

Comment:
I think you can afford to be more descriptive for the variable tmp. I understand what is echo but tmp is a bit ambiguous.